### DISCUSSION TITLE
Wireframe initial design discussion

### DISCUSSION THREAD

**Discussion by Swarnimashukla** [Swarnimashukla/Automatic-attendance-management-system#62] [General]
Hey all, 

Since Design is an iterative process and you must be confused whether to submit a design as **Final Design**  I have got a solution for you, before making a PR  you can share your designs here and take feedback from fellow participants, mentors and me. 

Happy Designing ❤❤

**Comment by ghost** [comment]
@Swarnimashukla  this is the logo design from my side we can play with the color themes as per the themes of our website. Below is the pdf file of the logo any suggestions are welcome.
[
[Roll Call.pdf](https://github.com/Swarnimashukla/Automatic-attendance-management-system/files/6136321/Roll.Call.pdf)
](url)

**Comment by Swarnimashukla** [comment]
Hey @garvitraj  it looks good  to me , can we have a version without rollcall text and a version with a  shade of blue color in background with white color for design part.

**Reply by ghost** [reply]
@Swarnimashukla yeah sure will make all the variation with the things u mention and just revert back to u asap
thanks
